packages/grafana-data/src/types/templateVars.ts

```diff
@@ -66,11 +65,10 @@ export interface AdHocVariableModel extends BaseVariableModel {
   baseFilters?: AdHocVariableFilter[];
 }
 
-export interface GroupByVariableModel extends BaseVariableModel {
+export interface GroupByVariableModel extends VariableWithOptions {
   type: 'groupby';
   datasource: DataSourceRef | null;
   groupByKeys: string[];
```

Comment:
I think we can also get rid of this, and instead rely on the `current` from `VariableWithOptions` type. Then we hace a way to have both key/value representation rather than value as with this proposal.